runtime/vm/createramclass.cpp

```diff
@@ -2803,14 +2803,7 @@ internalCreateRAMClassFromROMClass(J9VMThread *vmThread, J9ClassLoader *classLoa
 					module = javaVM->unamedModuleForSystemLoader;
 				}
 			} else {
-				if (J9ROMCLASS_IS_PRIMITIVE_TYPE(romClass)
-					|| (LOAD_LOCATION_PATCH_PATH == locationType)
-					|| (LOAD_LOCATION_MODULE == locationType)
-				) {
-					module = javaVM->javaBaseModule;
-				} else {
-					module = javaVM->unamedModuleForSystemLoader;
-				}
+				module = javaVM->javaBaseModule;
```

Comment:
Can you add a comment explaining why we chose this behaviour?  Once there's a comment here I'm good with the change.

Reply:
Comment added